### DISCUSSION TITLE
Updating on device config changes - new heater added to Eddi

### DISCUSSION THREAD

**Discussion by Cameleon99** [CJNE/ha-myenergi#393] [General]
I've made some changes to my Eddi and added a new Underfloor heater to the device, as well as the existing DHW immersion heater that is configured in HA. How do I get the plugin to recognise the new heater? It was previously called None, which shows up in HA, should it pick up the new Underfloor heater name automatically, or do I have to change it manually in HA? 

Also does HA allow you to change the priority between the heaters on an Eddi? I can see the priority of the Eddi vs Zappi, but not between the 2 heaters on the Eddi.

**Comment by Cameleon99** [comment]
Also is it possible to see which heater is currently being heated? I have some heaters set up inline in my UFH but I need to turn on the UFH pump which I can do via HA, but I need to know how to trigger it. At the moment I can only see a value of power that is being used by the Eddi Internal CT but not which heater?

**Reply by jmccrohan** [reply]
@Cameleon99 I've filed #396 which should address this.
